blocks/bricks/base.py

```diff
@@ -210,12 +210,14 @@ def __getattr__(self, name):
         if 'properties' in self.__dict__ and name in self.properties:
             return property(create_unbound_method(
                 getattr(self, self.properties[name]), self.brick))
-        raise AttributeError
+        raise AttributeError("Cannot find '%s' property in brick %s" %
+                             (name, str(self)))
 
     def __setattr__(self, name, value):
         # Mimic behavior of read-only properties
         if 'properties' in self.__dict__ and name in self.properties:
-            raise AttributeError("can't set attribute")
+            raise AttributeError("Can't set attribute '%s' in brick %s" %
```

Comment:
The current error message, `can't set attribute` is an exact copy of the error message that Python gives when trying to set a read-only property.